components/services/bat_ads/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "bat_ads",
   "display_name": "Bat Ads Service",
-  "sandbox_type": "utility",
+  "sandbox_type": "none",
```

Comment:
the process continually restarted on linux with the sandbox enabled. ledger service process also has sandbox disabled